history-of-the-quakers-v3/original/04-book-4-chapter-18.adoc

```diff
@@ -165,7 +169,10 @@ etc.
 
 The preamble recites,
 
-"`that many negroes have been suffered to remain in the meetings of the Quakers,
+[.embedded-content-document.legal]
+--
+
+that many negroes have been suffered to remain in the meetings of the Quakers,
```

Comment:
I wasn't sure if I should use ellipses here because it seems to start mid-sentence or maybe capitalize "that"?

Reply:
yeah, this is a little weird.  i think an ellipses would be good.  and also, there is a sentence in the middle that is not part of the document/quote. I think i'll tweak this a hair before merging.